demos/src/js/chartAuthoring-demo-overview.js

```diff
@@ -41,7 +57,10 @@ var demo = demo || {};
             storeModel: {
                 type: "fluid.modelComponent",
                 options: {
-                    gradeNames: ["fluid.prefs.cookieStore"],
+                    gradeNames: [
+                        "fluid.prefs.cookieStore",
+                        "fluid.prefs.cookieStore.writable"
+                    ],
```

Comment:
There doesn't seem to be anything stored in the cookie. I tried toggling the overview panel closed, changed values for the chart, and set the title and description. None of those are saved in the cookie and when a page is reloaded they all return to their default states.

Reply:
From looking at the code, it does look like it should only be storing the state of the OverViewPanel. I think the problem is actually with the model setup for this component. I've left a comment in the related section.